test/fixtures/challenges/challenges.js

```diff
@@ -0,0 +1,12 @@
+const { convertToSeconds } = require("../../../constants/time");
+
+module.exports = () => {
+  return [
+    {
+      title: "Sherlock and Anagrams",
+      level: "Easy",
+      start_date: parseInt(Date.now() / 1000),
+      end_date: parseInt(Date.now() / 1000) + convertToSeconds.DAYS_10, // After 10 days
+    },
+  ];
```

Comment:
Instead of doing mathematical calculations here, I am wondering if we can import a util function for our Date needs?

```js
getTimeNow()
getTimeAfter({ days: 10})
```

Also, why are we passing timestamp in a "<>_date" key?

Reply:
Done :+1: